core/src/main/scala/org/broadinstitute/dsde/rawls/submissions/SubmissionsService.scala

```diff
@@ -538,14 +538,7 @@ class SubmissionsService(
 
     for {
       _ <- requireComputePermission(workspaceName)
-
-      // getWorkflowFailureMode early because it does validation and better to error early
-      workflowFailureMode <- getWorkflowFailureMode(submissionRequest)
-
       workspaceContext <- getV2WorkspaceContextAndPermissions(workspaceName, SamWorkspaceActions.write)
-
-      submissionRoot = submissionRootPath(workspaceContext, submissionId)
```

Comment:
Moved this inline where it's needed, since there isn't really a failure case we need to deal with, or anything like that.